### PR TITLE
updated pytmv1 to 0.8.0

### PR DESCRIPTION
**Pull request by shaqnawe**
## Status
Ready/In Progress.

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
Updated pytmv1 package to 0.8.0.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -677,3 +677,7 @@ updates:
       - dependency-name: "pytmv1"
     schedule:
      interval: daily
+  - package-ecosystem: pip
```
Comment (JudahSchwartz) [inline .github/dependabot.yml:680]: please remove the entry for 
directory: /docker/pytmv1
note that you didnt add pytmv1 to be ignored so dependabot will autoupdate this now

Comment (shaqnawe) [reply .github/dependabot.yml]: done!

**Comment by JudahSchwartz** [comment]
@shaqnawe everything looks good. Please resolve conflicts and lets merge 🦾

**Comment by JudahSchwartz** [comment]
doesnt the t in pytmv stand for trend? Why rename?

**Comment by shaqnawe** [comment]
> doesnt the t in pytmv stand for trend? Why rename? also a manual pr to content will be needed because of the rename

I was running into an issue where the docker image was not updating correctly. Are there any other changes I need to make? I'm ready to make the PR for content, however I do not see the newest demisto image in dockerhub that I'm supposed to use with dockerimage in yml.

**Comment by JudahSchwartz** [comment]
@shaqnawe once this PR is merged you should see the image in dockerhub, nothing else needs to be done from your side, just to resolve the merge conflict so this can be merged
image

**Comment by shaqnawe** [comment]
> @shaqnawe once this PR is merged you should see the image in dockerhub, nothing else needs to be done from your side, just to resolve the merge conflict so this can be merged image

Do I need to go back to the original name to resolve this issue? it's complaining about a conflicting file but I don't even have the docker/pytmv1 dir anymore.

**Comment by JudahSchwartz** [comment]
@shaqnawe you shouldnt need to, but you can rebase this commit off master/ merge master into your branch and renaming should be fine. I hit the update with master button for you so you should be free to rename now if you want

**Comment by shaqnawe** [comment]
> @shaqnawe you shouldnt need to, but you can rebase this commit off master/ merge master into your branch and renaming should be fine. I hit the update with master button for you so you should be free to rename now if you want

@JudahSchwartz Anything else I need to do?

**Comment by JudahSchwartz** [comment]
@shaqnawe nothing else needs to be done here. Its recommended but not required to add to the verify.py

**Pull request merged**